src/apps/dashboard/routes/_asyncRoutes.ts

```diff
@@ -9,6 +9,7 @@ export const ASYNC_ADMIN_ROUTES: AsyncRoute[] = [
     { path: 'logs', type: AppType.Dashboard },
     { path: 'playback/trickplay', type: AppType.Dashboard },
     { path: 'plugins/:pluginId', page: 'plugins/plugin', type: AppType.Dashboard },
+    { path: 'tasks', page: 'scheduledtasks', type: AppType.Dashboard },
```

Comment:
Ideally the `path` should match the file layout in `routes` so let's just name the directory `tasks` also :+1: